diesel/src/associations/belongs_to.rs

```diff
@@ -27,8 +27,9 @@ impl<Parent, Child, Iter> GroupedBy<Parent> for Iter where
         let id_indices: HashMap<_, _> = parents.iter().enumerate().map(|(i, u)| (u.id(), i)).collect();
         let mut result = parents.iter().map(|_| Vec::new()).collect::<Vec<_>>();
```

Comment:
Yes. I have to arbitrarily do `Child: Clone` to write it that way though, since even though it's an empty vector we still only have `impl<T: Clone> Clone for Vec<T>`